ui/src/parsers/parseEntityRelationships.ts

```diff
@@ -88,6 +88,19 @@ const parseEntityRelationships = (objects: FeastRegistryType) => {
    });
  });
 
+  objects.streamFeatureViews?.forEach((fv) => {
+    links.push({
```

Comment:
you probably also want the batch source linked too right?

Reply:
yeah this is sth I haven't understand yet. looks like the `streamFeatureView` contains a `batchSource` and a `streamSource`, while the `streamSource` it self also contains a `batchSource`.
So is it guaranteed that each `streamFeatureView` will have this structure of data?

Reply:
Ah I think you'll actually be fine looks like. Might have been some artifact of before